packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -345,26 +351,29 @@ export const GiantTrackTile = ({
   }
 
   const renderListenCount = () => {
-    const shouldShow = !isUnlisted || fieldVisibility.play_count
+    const shouldShow =
+      isOwner || (!isPremium && (isUnlisted || fieldVisibility.play_count))
+
+    if (!shouldShow) {
+      return null
+    }
     return (
-      shouldShow && (
-        <div className={styles.listens}>
-          {listenCount === 0 ? (
-            <span className={styles.firstListen}>
-              Be the first to listen to this track!
+      <div className={styles.listens}>
```

Comment:
This looks weird because I pulled the `shouldShow` logic up to an early null return. Otherwise, this is nearly unchanged with the exception of hiding the "Be the first to listen" if you're the owner (will show 0 plays instead)